src/Utils/external/Dispose.ts

```diff
@@ -0,0 +1,42 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
```

Comment:
Could you let me know the reference link? :)

Reply:
The original code is located in https://github.com/microsoft/vscode-extension-samples/blob/main/custom-editor-sample/src/dispose.ts
This does not have copyright comment above so I referred vscode project of MS
https://github.com/microsoft/vscode

Reply:
I asked it because leaving reference link is better :)

```suggestion

// This file is from
// https://github.com/microsoft/vscode-extension-samples/blob/fdd3bb95ce8e38ffe58fc9158797239fdf5017f1/custom-editor-sample/src/dispose.ts

```

Reply:
I see. I will add both reference link and copy right comment in Dispose.ts!